functions/commonfn.php

```diff
@@ -357,6 +361,33 @@ function amuRegisterUser(){
 		//return message
 		return $newid;
 	}
+
+    /**
+     * Contacts central auth to determine if uun exists
+     *
+     * @param string $uun
+     *
+     * @return boolean
+     */
+    function is_valid_uun( $uun ) {
+        global $amu_ldap_config;
+        $ldapconn  = ldap_connect(  $amu_ldap_config['ldaphost'],  $amu_ldap_config['ldapport'] ) or wp_die( "Could not connect to {$amu_ldap_config['ldaphost']}", 200 );
```

Comment:
Can we rename `$ldapconn` to be `$ldap_conn`?

Reply:
done

Reply:
done

Reply:
Thank you!